cmd/main.go

```diff
@@ -142,6 +173,46 @@ Full docs can be found at github.com/aws/amazon-` + binName
 	}
 	resultsOutputFn := outputs.SimpleInstanceTypeOutput
 
+	cliSortDirections := []string{
+		sortAscending,
+		sortAsc,
+		sortDescending,
+		sortDesc,
+	}
+
+	sortingShorthandFlags := []string{
+		vcpus,
+		memory,
+		gpuMemoryTotal,
+		networkInterfaces,
+		spotPrice,
+		odPrice,
+		instanceStorage,
+		ebsOptimizedBaselineBandwidth,
+		ebsOptimizedBaselineThroughput,
+		ebsOptimizedBaselineIOPS,
+		gpus,
+		inferenceAccelerators,
+	}
+
+	sortingShorthandPaths := []string{
+		vcpuPath,
+		memoryPath,
+		gpuMemoryTotalPath,
+		networkInterfacesPath,
+		spotPricePath,
+		odPricePath,
+		instanceStoragePath,
+		ebsOptimizedBaselineBandwidthPath,
+		ebsOptimizedBaselineThroughputPath,
+		ebsOptimizedBaselineIOPS,
```

Comment:
Should this be `ebsOptimizedBaselineIOPSPath`?

Reply:
Yes it should be. Good catch. thank you!